src/NServiceBus.Metrics/MetricsFeature.cs

```diff
@@ -275,17 +277,32 @@ RawDataReporter CreateReporter(Action<Action<long, int>> setupProbe, string metr
         {
             var buffer = new RingBuffer();
 
-            var reporterHeaders = new Dictionary<string, string>(headers);
+            var reporterHeaders = new Dictionary<string, string>(headers)
+            {
+                {Headers.ContentType, contentType},
+                {MetricHeaders.MetricType, metricType}
+            };
+            
+            var dispatcher = builder.Build<IDispatchMessages>();
+            var address = new UnicastAddressTag(options.ServiceControlMetricsAddress);
 
-            reporterHeaders.Add(Headers.ContentType, contentType);
-            reporterHeaders.Add(MetricHeaders.MetricType, metricType);
+            Func<byte[], Task> sender = async body =>
+            {
+                var message = new OutgoingMessage(Guid.NewGuid().ToString(), reporterHeaders, body);
+
+                var operation = new TransportOperation(message, address);
+                try
+                {
+                    await dispatcher.Dispatch(new TransportOperations(operation), new TransportTransaction(), new ContextBag())
+                        .ConfigureAwait(false);
+                }
+                catch (Exception ex)
```

Comment:
Is it good enough that we swallow the exception and do not retry? Maybe we should extend [`RawDataReporter`](https://github.com/Particular/ServiceControl.Monitoring.Data/blob/master/src/ServiceControl.Monitoring.Data/RawDataReporter.cs#L90) with some retry policy? Either way sound like a separate PR.

Reply:
This is the same behavior as it was before. Also, it the ancient question: how to log/report when your logging/reporting fails. I see no other way than just reporting it for now.